homeassistant/components/vicare/__init__.py

```diff
@@ -87,20 +66,27 @@ def setup_vicare_api(hass: HomeAssistant, entry: ConfigEntry) -> None:
     vicare_api = vicare_login(hass, entry.data)
 
     device_config_list = get_supported_devices(vicare_api.devices)
+    if number_of_devices := len(device_config_list) > 1:
+        cache_duration = max(
+            DEFAULT_CACHE_DURATION, DEFAULT_CACHE_DURATION * number_of_devices
+        )
```

Comment:
Why is the `max` function used here? Clearly, multiplying a first positive integer with a second positive integer will always generate a larger value than the first positive integer, so either `max` is not needed, or there's some mistake in what is passed to `max`?

Reply:
You're right, in combination with the condition that no longer makes sense.

```suggestion
        cache_duration = DEFAULT_CACHE_DURATION * number_of_devices
```